fileModule.js

```diff
@@ -0,0 +1,30 @@
+var fileModule = (function() {
+  function handleFiles(config, callback) {
+    var files = document.querySelector(config.input).files; //variable, which shows how many files is loading
+    var file;
+    if (files) { 
+      for (var i = 0; i < files.length ; i++) {
+        file = files[i];
+        loadFile(file, config, callback);
+      }
+    }
+  }
+  function loadFile(file, config, callback) {
+    var reader = new FileReader(); //create new FileReader(), which lets read the content of files
+    if (!isFileAllowed(file, config.allowedFiles)) { //condition for images format

```

Comment:
nie umieszczalbym tego sprawdzania w tym miejscu. to dodawanie funkcjonalnosci nieco na sile. wolalbym zebys wystawila w api isFileAllowed (chociaz tez powinnas to nazwac nieco inaczej bo is... sugeruje mi property a nie method) i uzyla tego bezposrednio przy operacji pobierania plikow